wagtail/admin/userbar.py

```diff
@@ -84,7 +86,9 @@ class AccessibilityItem(BaseItem):
     axe_custom_checks = [
         {
             "id": "check-image-alt-text",
-            "options": {"pattern": "\\.(avif|gif|jpg|jpeg|png|svg|webp)$|_"},
+            "options": {
+                "pattern": f"\\.({'|'.join(get_allowed_image_extensions())})$|_"
```

Comment:
Suggestion: I'd recommend using `re.escape` here to avoid potential issues with the extensions. For example, someone accidentally prefixing the extension with a `.` could lead to odd behaviours.